Python/Product/Analysis/LanguageServer/Structures.cs

```diff
@@ -239,16 +240,18 @@ public struct Interpreter {
             public string version;
         }
         public Interpreter interpreter;
+
         /// <summary>
         /// Paths to search when attempting to resolve module imports.
         /// </summary>
-        public string[] searchPaths;
+        public string[] searchPaths = Array.Empty<string>();
+
         /// <summary>
         /// Secondary paths to search when resolving modules. Not supported by all
         /// factories. In generaly, only source files will be discovered, and their
         /// contents will be merged with the initial module.
         /// </summary>
-        public string[] typeStubSearchPaths;
+        public string[] typeStubSearchPaths = Array.Empty<string>();
```

Comment:
Do you think we should move `searchPaths` and `typeStubSearchPaths` into `LanguageServerSettings` so we can update them without restarting the process? (Not for this PR, unless it's easy)

Reply:
Eventually, yes. Just kept them here for simplicity for now. They don't change that often.